pkg/daemon/ceph/client/mirror.go

```diff
@@ -462,23 +462,33 @@ So the scenario looks like:
  6. Repeat the steps flipping source and destination to enable
     bi-directional mirroring
 */
-func CreateRBDMirrorBootstrapPeerWithoutPool(context *clusterd.Context, clusterInfo *ClusterInfo) ([]byte, error) {
-	fullClientName := getQualifiedUser(rbdMirrorPeerKeyringID)
+func CreateRBDMirrorBootstrapPeerWithoutPool(context *clusterd.Context, clusterInfo *ClusterInfo, rotateMirrorPeerKey bool) ([]byte, error) {
+	fullClientName := getQualifiedUser(RBDMirrorPeerKeyringID)
 	logger.Infof("create rbd-mirror bootstrap peer token %q", fullClientName)
 	key, err := AuthGetOrCreateKey(context, clusterInfo, fullClientName, rbdMirrorPeerCaps)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to create rbd-mirror peer key %q", fullClientName)
 	}
 	logger.Infof("successfully created rbd-mirror bootstrap peer token for cluster %q", clusterInfo.NamespacedName().Name)
 
+	if rotateMirrorPeerKey {
+		logger.Infof("rotating cephx key for rbd-mirror peer key %q", fullClientName)
```

Comment:
This will help disambiguate the log if multiple clusters are configured

```suggestion
		logger.Infof("rotating cephx key for rbd-mirror peer key %q in namespace %q", fullClientName, clusterInfo.Namespace)
```